lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -168,6 +177,16 @@ private void buildCompositionLayer() {
         this, Layer.Factory.newInstance(composition), composition.getLayers(), composition);
   }
 
+  private void applyColorFilters() {
+    if (compositionLayer == null) {
+      return;
+    }
+
+    for (ColorFilterData data : colorFilterData) {
```

Comment:
Just in case somebody calls setColorFilter in an update listener, can you make this a fori loop so it doesn't create an iterator object for each call?